content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -96,22 +118,49 @@ If you haven't already set up a cluster locally, run `minikube start` to create
    web       NodePort   10.104.133.249   <none>        8080:31637/TCP   12m
    ```
 
-1. Visit the Service via NodePort:
+1. Visit the Service via NodePort, using the [`minikube service`](https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-service-with-tunnel) command. Follow the instructions for your platform:
 
+   {{< tabs name="minikube_service" >}}
+   {{% tab name="Linux" %}}
+   
    ```shell
    minikube service web --url
    ```
-
+   
```

Comment:
nit: Extra whitespace should be removed.

Reply:
Resolved in the latest commit